utils/testdata/interfaceTest.sol

```diff
@@ -0,0 +1,4 @@
+interface test {
+     function guess(uint8 n) payable external;
+     function isComplete() view external returns (bool );
+}
```

Comment:
I'd expand the test case to also have >1 arguments in the function inputs and outputs, to showcase the `.join(", ")` working.

Also, nit notice how there's an extra space in the `returns (bool )`